repos/system_upgrade/common/actors/scancryptopolicies/tests/component_test_scancryptopolicies.py

```diff
@@ -0,0 +1,14 @@
+import distro
+
+from leapp.libraries.common.config import version
+from leapp.models import CryptoPolicyInfo
+
+
+def test_actor_execution(monkeypatch, current_actor_context):
+    target_version = int(distro.major_version()) + 1
```

Comment:
@Jakuje  this is wrong as it is affected by the system where the test is executed

Reply:
The problem is the actor test is now written in the way that it tests the actual files in the system. So to avoid this, I would have to mock also the reading of the files.

Reply:
well, in such a case it should mock also the reading of files. it cannot be dependent on anything on the system where the test is executed.

Reply:
Should be also resolved.